src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -22,14 +22,16 @@ import org.rust.lang.core.types.ty.TyUnknown
 import org.rust.lang.core.types.type
 import javax.swing.Icon
 
+val FUNCTION_TEST_REGEX = Regex(".*test.*")
```

Comment:
Unfortunately, it's too general and makes plugin provide test gutter icon for non-test attributes like https://github.com/tokio-rs/tokio/blob/71c47fabf4a8a450c3b41d58de304a6a49fb4061/tokio-macros/src/lib.rs#L59
Let's restrict it a little bit. For example, `^[\w:]*test.*` or something similar.
And let's add test(s) to check that we don't provide false run icon for some basic cases like I've pointed above

Reply:
Good catch.